python/ray/data/aggregate.py

```diff
@@ -1486,11 +1499,15 @@ def __init__(
             log_capacity: Base 2 logarithm of the maximum size of the internal hash map.
                 Higher values increase accuracy but use more memory. Defaults to 15.
             alias_name: The name of the aggregate. Defaults to None.
+            encode_lists: If ``True``, encode list elements.  If ``False``, encode
+                whole lists (i.e., the entire list is considered as a single object).
+                ``False`` by default.
         """
 
         self.k = k
         self._log_capacity = log_capacity
         self._frequent_strings_sketch = self._require_datasketches()
+        self.encode_lists = encode_lists
```

Comment:
Nit: For consistency with the other attributes?
```suggestion
        self.encode_lists = encode_lists
```